CHANGELOG.md

```diff
@@ -1,6 +1,9 @@
 ## Cairo-VM Changelog
 
 #### Upcoming Changes
+
+* feat: wrap big variants of `HintError` in `Box` [#1193](https://github.com/lambdaclass/cairo-rs/pull/1193)
```

Comment:
This is also `BREAKING`. We should mention that in the changelog.

Reply:
Added